### PR TITLE
Replace auto generated column headers with a localized version

### PR DESCRIPTION
**Pull request by HenryEstberg**
Small fix to replace auto generated column headers with a localized version that can be translated

### CODE REVIEW

File: pwiz_tools/Skyline/FileUI/ImportTransitionListColumnSelectDlg.cs [hunk 1]
```diff
@@ -108,7 +108,11 @@ private void DisplayData()
             {
                 for (var i = 0; i < numColumns; i++)
                 {
-                    dataGrid.Columns[i].ToolTipText =
+                    // In the case when we don't have user provided headers, we still want localized headers that can be translated,
+                    // this replaces the autogenerated strings with a localized version
+                    dataGrid.Columns[i].HeaderText = string.Format(Resources.ImportTransitionListColumnSelectDlg_DisplayData_Column_) + (i+1).ToString();
+
```
Comment (bspratt) [inline pwiz_tools/Skyline/FileUI/ImportTransitionListColumnSelectDlg.cs:114]: Probably cleaner and more general to do this:
string,Format("Column {0}", i+1)

That way if the local language has a convention like "3 column, this is" it still works.

File: pwiz_tools/Skyline/Skyline.sln.DotSettings [hunk 2]
```diff
@@ -143,6 +143,7 @@
 	<s:String x:Key="/Default/CodeStyle/Naming/CSharpNaming/PredefinedNamingRules/=PrivateInstanceFields/@EntryIndexedValue">&lt;Policy Inspect="True" Prefix="_" Suffix="" Style="aaBb" /&gt;</s:String>
 	<s:String x:Key="/Default/CodeStyle/Naming/CSharpNaming/PredefinedNamingRules/=PrivateStaticFields/@EntryIndexedValue">&lt;Policy Inspect="True" Prefix="_" Suffix="" Style="aaBb" /&gt;</s:String>
 	<s:String x:Key="/Default/CodeStyle/Naming/CSharpNaming/PredefinedNamingRules/=PrivateStaticReadonly/@EntryIndexedValue">&lt;Policy Inspect="True" Prefix="" Suffix="" Style="AA_BB" /&gt;</s:String>
+	<s:Boolean x:Key="/Default/Environment/SettingsMigration/IsMigratorApplied/=JetBrains_002EdotCover_002EIde_002ECore_002EFilterManagement_002EModel_002ESolutionFilterSettingsManagerMigrateSettings/@EntryIndexedValue">True</s:Boolean>
```
Comment (bspratt) [inline pwiz_tools/Skyline/Skyline.sln.DotSettings:146]: Let's not leave this in - something to do with the ReSharper version, might not be good everywhere.

**Review by bspratt — CHANGES_REQUESTED** [review]
Just a couple of tweaks needed. Get those in and I'll look again.

File: pwiz_tools/Skyline/FileUI/ImportTransitionListColumnSelectDlg.cs (hunk 1 above)
Comment (bspratt) [reply pwiz_tools/Skyline/FileUI/ImportTransitionListColumnSelectDlg.cs:114]: (So they'd swap in a string like "{0} column, this is")

File: pwiz_tools/Skyline/Skyline.sln.DotSettings (hunk 2 above)
Comment (HenryEstberg) [reply pwiz_tools/Skyline/Skyline.sln.DotSettings]: I'm not quite sure what is causing this to be updated, but whenever I manually remove it and then open VS, this line gets added again. I removed it for the most recent commit, but I'd really like to find whatever setting is doing this so its not an issue in the future.

File: pwiz_tools/Skyline/FileUI/ImportTransitionListColumnSelectDlg.cs (hunk 1 above)
Comment (HenryEstberg) [reply pwiz_tools/Skyline/FileUI/ImportTransitionListColumnSelectDlg.cs:114]: Should be fine now

File: pwiz_tools/Skyline/Properties/Resources.resx [hunk 3]
```diff
@@ -5568,7 +5568,7 @@ If you prefer you can choose to report anonymously.</value>
   <data name="ExportMethodDlg_comboInstrument_SelectedIndexChanged_Export_of_DIA_isolation_lists_is_not_yet_supported_for__0__" xml:space="preserve">
     <value>Export of DIA isolation lists is not yet supported for {0}.</value>
   </data>
-  <data name="MassListRowReader_NextRow_No_peptide_sequence_column_specified">
+  <data name="MassListRowReader_NextRow_No_peptide_sequence_column_specified" xml:space="preserve">
```
Comment (bspratt) [inline pwiz_tools/Skyline/Properties/Resources.resx:5571]: Some unrelated changes have wandered in here, that's not good

**Comment by HenryEstberg** [comment]
Yeah the gitignore thing was unintentional, reverting now.

**Review by bspratt — APPROVED** [review]
OK, nice minimal change set, and you've given me a live demo. This is good to go, merge it to master then delete this branch, as is our convention.
